src/gam.py

```diff
@@ -11235,14 +11265,25 @@ def doPrintLicenses(returnFields=None, skus=None):
   writeCSVfile(csvRows, titles, u'Licenses', todrive)
 
 RESCAL_DFLTFIELDS = [u'id', u'name', u'email',]
-RESCAL_ALLFIELDS = [u'id', u'name', u'email', u'description', u'type',]
+RESCAL_ALLFIELDS = [u'id', u'name', u'email', u'description', u'type', u'buildingid', u'category', u'capacity', u'features', u'floor', u'floorsection', u'generatedresourcename', u'uservisibledescription',]
```

Comment:
This is fine for now but ultimately would like to start handling these fields more dynamically based on discovery file like print buildings code does. If we were already doing that here then no GAM code change would have been needed after Google added the new fields to resource calendars.